doc/api/documentation.markdown

```diff
@@ -20,6 +20,12 @@ Every `.html` and `.json` file is generated based on the corresponding
 documentation is generated using the `tools/doc/generate.js` program.
 The HTML template is located at `doc/template.html`.
 
+
+If you find a error in this documentation, please
+[submit an issue](https://github.com/nodejs/node/issues/new),
+or see [the contributing guide](https://github.com/nodejs/node/blob/master/CONTRIBUTING.md)
```

Comment:
the links here can actually be moved to the end of the document... in-line simply do `[submit an issue][]` and `[the contributing guide][]`, then at the bottom of the documentation.markdown file, add:

```
[submit an issue]: https://github.com/nodejs/node/issues/new
[the contributing guide]: https://github.com/nodejs/node/blob/master/CONTRIBUTING.md
```

doing so would ensure that the text itself will line wrap correctly and will be easier to read later if changes need to be made.